server/middleware/Middleware.js

```diff
@@ -21,6 +22,24 @@ require('dotenv').config();
 
 /** middleware class */
 class Middleware {
+  /**
+   *
+   * @param {object} req
```

Comment:
The should be a description for JSDOCs, and also there should be space between concerns (desc, params and returns)